File: program/program/print-elements-of-an-array/PrintElementsOfAnArray.java
public class PrintElementsOfAnArray {
    public static void main(String[] args) {
        int [] arr = {1,2,3,4,5};
        printArrayElements(arr);
    }
    static void printArrayElements(int [] nums){
        for(int element: nums){
            System.out.print(element + " ");
        }
    }
}

File: program/program/print-elements-of-an-array/print_elements_of_an_array.r
# Function to print elements in an array
printArrayElements <- function(arr) {
  n <- length(arr)
  
  for (i in 1:n) {
    cat(" ", arr[i])
  }
}

# Function call
myArray <- c(1, 2, 3, 4, 5)
printArrayElements(myArray)
